### PR TITLE
enabling e2e tests on both chrome and firefox

### PR DESCRIPTION
**Pull request by mozrokafor**
Enabling e2e tests for chrome and firefox thus reducing amount of manual effort for QA and getting closer to faster deployments

# References: 
Jira: [MNTOR-2165](https://mozilla-hub.atlassian.net/browse/MNTOR-2165) and [MNTOR-2164](https://mozilla-hub.atlassian.net/browse/MNTOR-2164)

[MNTOR-2165]: https://mozilla-hub.atlassian.net/browse/MNTOR-2165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MNTOR-2164]: https://mozilla-hub.atlassian.net/browse/MNTOR-2164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mansaj** [comment]
Having FF back would be awesome! Thanks for taking on this work @mozrokafor !

File: playwright.config.js [hunk 1]
```diff
@@ -104,10 +103,10 @@ export default defineConfig({
...
-    port: 6060,
-    // Building the app can take some time:
-    timeout: 1_800_000,
-  }
+  // webServer: {
+  //   command: 'npm run build; npm start',
+  //   port: 6060,
+  //   // Building the app can take some time:
+  //   timeout: 1_800_000,
+  // }
```
Comment (pdehaan) [inline playwright.config.js:111]: Q: Do we not need this anymore, or is this just a temporary commenting-out?

File: playwright.config.js [hunk 2]
```diff
@@ -72,12 +72,11 @@ export default defineConfig({
...
-
-    // {
-    //   name: 'firefox',
-    //   use: { ...devices['Desktop Firefox'] }
-    // }
+    },
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] }
+    },
```
Comment (pdehaan) [inline playwright.config.js:79]: Noice!
Does this mean you'll need to regen a bunch of baseline images and have them potentially be browser specific?

File: playwright.config.js (hunk 1 above)
Comment (mozrokafor) [reply playwright.config.js]: ah accident

File: playwright.config.js (hunk 2 above)
Comment (pdehaan) [reply playwright.config.js:79]: Define "env". Like, you mean Stage will be different than Heroku and Prod?

**Review by pdehaan — APPROVED** [review]
LGTM. CI is green.
You said you still have baselines and "envs" for future TODOs, which is cool.
Pretty sure Playwright doesn't support other channels (last I checked), so probably no luck trying in Beta/Nightly releases.

**Pull request merged**